shared/utils/generate-from-templates.py

```diff
@@ -15,6 +15,7 @@
 from create_mounts import MountsGenerator
 from create_mount_options import MountOptionsGenerator
 from create_package_installed import PackageInstalledGenerator
+from create_package_required_version_installed import PackageRequiredVersionInstalledGenerator
```

Comment:
merge into create_package_installed rather than a separate template just for required package versions.